lib/internal/test_runner/mock/fake_timers.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+const {
+  DateNow,
+  globalThis,
+} = primordials;
+
+const PriorityQueue = require('internal/priority_queue');
+
+function compareTimersLists(a, b) {
+  return (a.runAt - b.runAt) || (a.id - b.id);
+}
+
+function setPosition(node, pos) {
+  node.priorityQueuePosition = pos;
+}
+class Timers {
+  #currentTimer = 1;
+  constructor() {
+    this.timers = new PriorityQueue(compareTimersLists, setPosition);
+
+    this.setTimeout = this.#createTimer.bind(this, false);
+    this.clearTimeout = this.#clearTimer.bind(this);
+    this.setInterval = this.#createTimer.bind(this, true);
+    this.clearInterval = this.#clearTimer.bind(this);
+  }
+
+  #createTimer(isInterval, callback, delay, ...args) {
+    const timerId = this.#currentTimer++;
+    this.timers.insert({
+      id: timerId,
+      callback,
+      runAt: DateNow() + delay,
+      interval: isInterval,
+      args,
+    });
+
+    return timerId;
+  }
+
+  #clearTimer(position) {
+    this.timers.removeAt(position);
+  }
+
+}
+
+let realSetTimeout;
+let realClearTimeout;
+let realSetInterval;
+let realClearInterval;
+
+class FakeTimers {
+  constructor() {
+    this.fakeTimers = {};
+    this.isEnabled = false;
+    this.now = DateNow();
+  }
+
+  tick(time = 0) {
```

Comment:
```suggestion
  tick(time =1) {
```
shouldn't the default tick 1ms?